### ISSUE TITLE
LOSS is shown as Nan.

### ISSUE THREAD

**Issue by ccstudyq** [yassouali/pytorch-segmentation#138]
Hi, I'm a newbie in semantic segmentation, during the training process, I found that the loss keeps showing as Nan.I would like to know what is the reason, I hope I can get help to solve it.

**Comment by yassouali** [comment]
Hi @ccstudyq 

Maybe try to make sure your targets are correct

**Comment by yassouali** [comment]
feel free to reopen / tag me if you still have a problem
